site/_config.yml

```diff
@@ -64,6 +65,11 @@ defaults:
     values:
       toc: true
       sitemap: true
+  -
```

Comment:
This sets the default `category` frontmatter variable for all pages to be `documentation`.